src/include/sof/lib/alloc.h

```diff
@@ -105,9 +109,10 @@ struct mm {
 /* heap allocation and free */
 void *_malloc(int zone, uint32_t caps, size_t bytes);
 void *_zalloc(int zone, uint32_t caps, size_t bytes);
-void *_balloc(int zone, uint32_t caps, size_t bytes);
+void *_balloc(int zone, uint32_t caps, size_t bytes, uint32_t alignment);
```

Comment:
This parameter is usually called 'align'.

Reply:
Lets add my personal spin to it shall we ? :D Or is it imperative we change it to align ?

Reply:
I want to keep the code consistent with most of the ways we are naming alignment which is align. Anyhow, I encourge creativity and personal spins so if you get 2 approvals you can submit the code.